### PR TITLE
Fix ShuffleNet ONNX export

### PR DESCRIPTION
**Pull request by justanhduc**
When exporting the current model, there's an error as follows
```
SymbolicValueError: Unsupported: ONNX export of operator prim::ConstantChunk, 
unknown dimension size. Please feel free to request support or submit a pull 
request on PyTorch GitHub: https://github.com/pytorch/pytorch/issues  [Caused by
the value '406 defined in (%406 : Float(*, *, *, *, strides=[249856, 1024, 32, 
1], requires_grad=1, device=cpu) = onnx::Reshape(%403, %405), scope: 
__main__.OnnxModel::/models.ShuffleNetv2x2::net/torchvision.models.shufflenetv2.
ShuffleNetV2::model/torch.nn.modules.container.Sequential::stage2/torchvision.mo
dels.shufflenetv2.InvertedResidual::stage2.0 # 
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torchvision/models
/shufflenetv2.py:38:0
)' (type 'Tensor') in the TorchScript graph. The containing node has kind 
'onnx::Reshape'.] 
    (node defined in 
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torchvision/models
/shufflenetv2.py(38): channel_shuffle
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torchvision/models
/shufflenetv2.py(99): forward
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/nn/modules/m
odule.py(1488): _slow_forward
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/nn/modules/m
odule.py(1501): _call_impl
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/nn/modules/c
ontainer.py(217): forward
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/nn/modules/m
odule.py(1488): _slow_forward
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/nn/modules/m
odule.py(1501): _call_impl
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torchvision/models
/shufflenetv2.py(157): _forward_impl
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torchvision/models
/shufflenetv2.py(166): forward
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/nn/modules/m
odule.py(1488): _slow_forward
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/nn/modules/m
odule.py(1501): _call_impl
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/nn/modules/c
ontainer.py(217): forward
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/nn/modules/m
odule.py(1488): _slow_forward
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/nn/modules/m
odule.py(1501): _call_impl
/home/ubuntu/workspace/ducna19/projects/yolox-bytetrack-count/carseat_cls/pt2onn
x.py(36): forward
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/nn/modules/m
odule.py(1488): _slow_forward
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/nn/modules/m
odule.py(1501): _call_impl
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/jit/_trace.p
y(118): wrapper
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/jit/_trace.p
y(127): forward
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/nn/modules/m
odule.py(1501): _call_impl
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/jit/_trace.p
y(1268): _get_trace_graph
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/onnx/utils.p
y(893): _trace_and_get_graph_from_model
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/onnx/utils.p
y(989): _create_jit_graph
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/onnx/utils.p
y(1113): _model_to_graph
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/onnx/utils.p
y(1548): _export
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torch/onnx/utils.p
y(506): export
/home/ubuntu/workspace/ducna19/projects/yolox-bytetrack-count/carseat_cls/pt2onn
x.py(69): main
/home/ubuntu/workspace/ducna19/projects/yolox-bytetrack-count/carseat_cls/pt2onn
x.py(113): <module>
)

    Inputs:
        #0: 403 defined in (%403 : Float(*, *, *, *, *, strides=[249856, 2048, 
1024, 32, 1], requires_grad=1, device=cpu) = onnx::Transpose[perm=[0, 2, 1, 3, 
4]](%402), scope: 
__main__.OnnxModel::/models.ShuffleNetv2x2::net/torchvision.models.shufflenetv2.
ShuffleNetV2::model/torch.nn.modules.container.Sequential::stage2/torchvision.mo
dels.shufflenetv2.InvertedResidual::stage2.0 # 
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torchvision/models
/shufflenetv2.py:35:0
    )  (type 'Tensor')
        #1: 405 defined in (%405 : int[] = prim::ListConstruct(%382, %404, %391,
%396), scope: 
__main__.OnnxModel::/models.ShuffleNetv2x2::net/torchvision.models.shufflenetv2.
ShuffleNetV2::model/torch.nn.modules.container.Sequential::stage2/torchvision.mo
dels.shufflenetv2.InvertedResidual::stage2.0
    )  (type 'List[int]')
    Outputs:
        #0: 406 defined in (%406 : Float(*, *, *, *, strides=[249856, 1024, 32, 
1], requires_grad=1, device=cpu) = onnx::Reshape(%403, %405), scope: 
__main__.OnnxModel::/models.ShuffleNetv2x2::net/torchvision.models.shufflenetv2.
ShuffleNetV2::model/torch.nn.modules.container.Sequential::stage2/torchvision.mo
dels.shufflenetv2.InvertedResidual::stage2.0 # 
/home/ubuntu/miniconda3/envs/pt20/lib/python3.8/site-packages/torchvision/models
/shufflenetv2.py:38:0
    )  (type 'Tensor')

Process finished with exit code 1
```
Replacing `-1` by `num_channels` fixed this problem.

cc @neginraoof @justinchuby

### CODE REVIEW

**Review by NicolasHug — APPROVED** [review]
Thanks for the PR @justanhduc .

LGTM when green. We'll need you to approve the CLA before merging.

**Comment by justanhduc** [comment]
@NicolasHug There are some failed tests. Is there anything I should do?

**Comment by NicolasHug** [comment]
They're unrelated, I'll merge the PR now. Thanks for the fix!

**Pull request merged**
